### PR TITLE
Misc improvements to LUT texture generation

### PR DESCRIPTION
**Pull request by cancan101**

### CODE REVIEW

File: src/helpers/helpers.lut.js [hunk 1]
```diff
@@ -45,16 +45,16 @@ export default class HelpersLut {
...
     return canvasContainer;
   }
 
   createCanvas() {
     let canvas = document.createElement('canvas');
-    canvas.height = 16;
+    canvas.height = 1;
     canvas.width = 256;
+    canvas.style.width = '256px';
+    canvas.style.height = '16px';
```
Comment (NicolasRannou) [inline src/helpers/helpers.lut.js]: shouldn't `canvas.height = 1;` and `canvas.style.height = '16px';` or do they have different meanings (i.e. `canvas.height` is the size of the actual canvas and `canvas.style.height` the representation of the same canvas in the DOM?)

**Comment by NicolasRannou** [comment]
Looking good, `height` was arbitrary as we were directly showing the canvas from this helper to the user. It makes sense to have 1 canvas for "presentation" (i.e. if someone wants to display the LUT in the DOM) and a canvas for pure computations (to be passed to shaders.)

FYI the `discrete` option may be irrelevant now as we now have a segmentation lut helper (https://github.com/FNNDSC/ami/blob/dev/src/helpers/helpers.segmentationlut.js), altough its purpose is a bit different -

See comment before merge!
Thanks!

File: src/helpers/helpers.lut.js (hunk 1 above)
Comment (cancan101) [reply src/helpers/helpers.lut.js]: Yes, height is the actual size of the pixel when painting to it, whereas the style of the Dom element is how it gets rendered.

**Comment by cancan101** [comment]
More generally this helper does two things : 1. creates a texture for rendering from the shaders and 2. Renders a swatch into a passed in Dom node. It might make sense to separate these concepts.

**Comment by NicolasRannou** [comment]
Agree - I think it makes sense to have helper to create textures for the shaders and another helper that would take any texture and generate its DOM reprensetation

**Pull request merged**